### PR TITLE
Simplified Chinese and Traditional Chinese localization updates

### PR DESCRIPTION
**Pull request by sbzlzh**

### CODE REVIEW

**Comment by saibotk** [comment]
@TheOnly8Z or @TEGTainFan would you mind taking a look?

**Comment by sbzlzh** [comment]
TheOnly8Z does not seem to be involved, TEGTainFan is probably busy

File: lua/terrortown/lang/zh_hans.lua [hunk 1]
```diff
@@ -23,24 +23,24 @@ L.innocents = "无辜阵营"
...
 L.win_traitor = "叛徒取得了胜利！"
 L.win_innocent = "叛徒们被击败了！"
-L.win_nones = "蜜蜂们胜利了！（平局）"
+L.win_nones = "无人胜出！(平局)"
 L.win_showreport = "来看一下 {num} 秒的回合总结吧！"
 
---L.limit_round = "Round limit reached. The next map will load soon."
---L.limit_time = "Time limit reached. The next map will load soon."
+L.limit_round = "达到回合数上限. 下一张地图即将加载."
+L.limit_time = "已达到时限. 下一张地图即将加载."
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:43]:
```suggestion
L.limit_time = "达到时间限制。即将加载下一张地图。"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 2]
```diff
@@ -23,24 +23,24 @@ L.innocents = "无辜阵营"
...
 L.win_time = "时间用尽，叛徒失败了。"
 L.win_traitor = "叛徒取得了胜利！"
 L.win_innocent = "叛徒们被击败了！"
-L.win_nones = "蜜蜂们胜利了！（平局）"
+L.win_nones = "无人胜出！(平局)"
 L.win_showreport = "来看一下 {num} 秒的回合总结吧！"
 
---L.limit_round = "Round limit reached. The next map will load soon."
---L.limit_time = "Time limit reached. The next map will load soon."
+L.limit_round = "达到回合数上限. 下一张地图即将加载."
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:42]:
```suggestion
L.limit_round = "达到回合限制。即将加载下一张地图。"
```

**Comment by TheOnly8Z** [comment]
Looks like there are some incorrect punctuation usage - parenthesizes, commas and periods should be full-width, e.g:

> 测试，请用全角标点（像这样）。

Instead of 

> 测试, 别用半角标点 (像这样).

I will start a code review and make amendments.

File: lua/terrortown/lang/zh_hans.lua [hunk 3]
```diff
@@ -296,7 +296,7 @@ L.c4_status_disarmed = "拆除"
 
 -- Visualizer
 L.vis_name = "显像器"
-L.vis_hint = "按下 {usekey} 键捡起它（仅限侦探）。"
+L.vis_hint = "按下 {usekey} 键捡起它(仅限侦探)。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:299]: Should not use half-width punctuation.

File: lua/terrortown/lang/zh_hans.lua [hunk 4]
```diff
@@ -402,8 +402,8 @@ L.dna_notfound = "目标上没有DNA样本。"
 L.dna_limit = "已达最大采集额度，请先移除旧样本。"
 L.dna_decayed = "凶手的DNA样本已经消失。"
 L.dna_killer = "成功采集到凶手的DNA样本！"
---L.dna_duplicate = "Match! You already have this DNA sample in your scanner."
-L.dna_no_killer = "DNA样本无法检索（凶手已离线？）"
+L.dna_duplicate = "匹配! 你的扫描仪里已经有这个DNA样本了."
+L.dna_no_killer = "DNA样本无法检索(凶手已离线？)"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:406]: Should not use half-width punctuation.

File: lua/terrortown/lang/zh_hans.lua [hunk 5]
```diff
@@ -483,9 +483,9 @@ L.karma_min = "滥杀者"
 
 -- TargetID misc
 L.corpse = "尸体"
-L.corpse_hint = "按下 {usekey} 来搜索，用 {walkkey} + {usekey} 进行无声搜索。"
+L.corpse_hint = "按下 [{usekey}] 来搜索，用 [{walkkey} + {usekey}] 进行无声搜索。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:486]: Half-width is fine here since it is brackets around a key.

File: lua/terrortown/lang/zh_hans.lua [hunk 6]
```diff
@@ -1459,21 +1459,19 @@ L.button_start_entspawn_edit = "开始生成点编辑"
...
+应该注意的是，这个生成系统使用动态生成。这将使武器生成系统更加有趣，因为它不再定义一个特定的武器。而是定义一种武器类型。例如，现在不是生成TTT自带的霰弹枪，而是生成被分类成霰弹枪的武器，任何定义为霰弹枪的武器都可以生成。每个武器的生成类型可以在装备编辑器中设置，这使得任何武器都可以在地图上生成，或者禁用某些默认武器。
 
---Keep in mind that many changes only take effect after a new round has started.]]
---L.help_spawn_editor_enable = "On some maps it might be advised to use the original spawns found on the map without replacing them with the dynamic system. Changing this option below only affects the currently active map, so the dynamic system will still be used for every other map."
+请记住，更改只有在新的一轮开始后才会生效。]]
+L.help_spawn_editor_enable = "在某些地图上，可能会建议使用在地图自带的原始生成点，而不用动态系统来取代它们。禁用这个复选框只对当前活动地图禁用。其他地图仍将使用动态系统。"
 L.help_spawn_editor_hint = "提示：要离开生成编辑器,重新打开游戏模式菜单。"
 L.help_spawn_editor_spawn_amount = [[
 目前在这张地图上有{weapon}武器生成，{ammo}弹药生成和{player}玩家生成。
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1477]:
```suggestion
目前在这张地图上有 {weapon} 个武器生成点，{ammo} 个弹药生成点和 player} 个玩家生成点。
```

File: lua/terrortown/lang/zh_hans.lua [hunk 7]
```diff
@@ -1512,25 +1510,25 @@ L.header_playermodels_general = "通用玩家模型设置"
...
+L.help_use_custom_models = [[
+默认情况下，只有CS起源版凤凰战士的模型被分配给所有玩家，然而，如果启用这个选项，将使用玩家模型库，启用此设置后，每个玩家将被分配到相同的玩家模型，但这些模型将从模型库中选择。
 
---This selection of models can be extended by installing more player models.]]
+模型选择可通过安装更多的玩家模型来扩展。]]
 
 -- 2021-10-06
 L.menu_server_addons_title = "服务器插件"
---L.menu_server_addons_description = "Server-wide admin only settings for addons."
+L.menu_server_addons_description = "仅管理员可以对附加插件进行设置。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1531]: Incorrect context.

```suggestion
L.menu_server_addons_description = "服务器端管理员专用的插件设置。"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 8]
```diff
@@ -1743,45 +1741,45 @@ L.label_prop_carrying_force = "Prop推进力"
...
+L.label_armor_value = "护甲物品所赋予的护甲"
 L.label_armor_damage_block_pct = "护甲承受的伤害百分比"
 L.label_armor_damage_health_pct = "玩家承受的伤害百分比"
 L.label_armor_enable_reinforced = "启用强化护甲"
 L.label_armor_threshold_for_reinforced = "强化护甲阈值"
 L.label_sherlock_mode = "启用侦探模式"
 L.label_highlight_admins = "突出服务器管理员"
 L.label_highlight_dev = "突出显示TTT2开发者"
---L.label_highlight_vip = "Highlight TTT2 VIP"
+L.label_highlight_vip = "突出显示TTT2 VIP"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1782]:
```suggestion
L.label_highlight_vip = "高亮显示 TTT2 VIP"
```

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by sbzlzh** [comment]
ok

File: lua/terrortown/lang/zh_hans.lua [hunk 9]
```diff
@@ -24,7 +24,7 @@ L.traitors = "叛徒阵营"
 
 -- Round status messages
 L.round_minplayers = "没有足够的玩家来开始新的回合..."
-L.round_voting = "投票进行中，新的回合将推迟到 {num} 秒后开始..."
+L.round_voting = "投票进行中，新的回合将推迟到 {num} 秒后开始... "
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:27]: extra space

File: lua/terrortown/lang/zh_hans.lua [hunk 10]
```diff
@@ -1058,11 +1058,11 @@ L.submenu_administration_hud_title = "HUD 设置"
...
 
 L.help_color_desc = "此选项启用后，准星和目标高亮的外框会显示一个全局通用的颜色"
-L.help_scale_factor = "这个比例影响所有界面大小 (HUD, vgui 和目标高亮)。屏幕分辨率修改后会自动更新这个比例。改变此比例会重置 HUD！"
+L.help_scale_factor = "这个比例影响所有界面大小（HUD, vgui 和目标高亮）。屏幕分辨率修改后会自动更新这个比例。改变此比例会重置 HUD！"
 L.help_hud_game_reload = "这个 HUD 当前无法使用，必须先重新加载游戏。"
 L.help_hud_special_settings = "以下是当前 HUD 的专用设定。"
-L.help_vskin_info = "VSkin(vgui 皮肤)是应用在所有菜单界面(包括这个)的皮肤。皮肤可以用一个简单的 Lua 脚本创建，并能改变颜色，大小等参数。"
-L.help_targetid_info = "目标高亮(TargetID)是显示在实体和玩家下面的信息。可以在通用菜单启用全局颜色。"
+L.help_vskin_info = "VSkin（vgui 皮肤）是应用在所有菜单界面(包括这个)的皮肤。皮肤可以用一个简单的 Lua 脚本创建，并能改变颜色，大小等参数。"
+L.help_targetid_info = "目标高亮TargetID是显示在实体和玩家下面的信息。可以在通用菜单启用全局颜色。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1065]: Probably better to keep the parenthesizes here.

**Comment by TheOnly8Z** [comment]
Looks good, just a a few small things and we can merge.

**Comment by sbzlzh** [comment]
All right

**Comment by sbzlzh** [comment]
@TEGTainFan Suggest you help to modify the Traditional Chinese

File: lua/terrortown/lang/zh_hans.lua [hunk 11]
```diff
@@ -1058,11 +1058,11 @@ L.submenu_administration_hud_title = "HUD 设置"
 L.submenu_administration_randomshop_title = "随机商店"
 
 L.help_color_desc = "此选项启用后，准星和目标高亮的外框会显示一个全局通用的颜色"
-L.help_scale_factor = "这个比例影响所有界面大小 (HUD, vgui 和目标高亮)。屏幕分辨率修改后会自动更新这个比例。改变此比例会重置 HUD！"
+L.help_scale_factor = "这个比例影响所有界面大小（HUD, vgui 和目标高亮）。屏幕分辨率修改后会自动更新这个比例。改变此比例会重置 HUD！"
 L.help_hud_game_reload = "这个 HUD 当前无法使用，必须先重新加载游戏。"
 L.help_hud_special_settings = "以下是当前 HUD 的专用设定。"
 L.help_vskin_info = "VSkin（vgui 皮肤）是应用在所有菜单界面（包括这个）的皮肤。皮肤可以用一个简单的 Lua 脚本创建，并能改变颜色，大小等参数。"
-L.help_targetid_info = "目标高亮（TargetID）是显示在实体和玩家下面的信息。可以在通用菜单启用全局颜色。"
+L.help_targetid_info = "目标高亮(TargetID)是显示在实体和玩家下面的信息。可以在通用菜单启用全局颜色。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1065]:
```suggestion
L.help_targetid_info = "目标高亮（TargetID）是显示在实体和玩家下面的信息。可以在通用菜单启用全局颜色。"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 12]
```diff
@@ -1743,45 +1741,45 @@ L.label_prop_carrying_force = "Prop推进力"
...
 L.label_postround_dm = "回合结束后启用死亡竞赛"
 L.label_spectator_chat = "启用观察者与大家聊天的功能"
 L.label_lastwords_chatprint = "如果在打字时被杀，则发出最后一句话至聊天室"
 L.label_identify_body_woconfirm = "不按'确认'按钮识别尸体"
 L.label_announce_body_found = "宣布发现了一具尸体"
 L.label_confirm_killlist = "宣布确认尸体时，该尸体的击杀名单"
---L.label_inspect_detective_only = "Limit corpse search to policing roles only"
---L.label_confirm_detective_only = "Limit corpse confirmation to policing roles only"
+L.label_inspect_detective_only = "将搜尸的范围限定为只限于警察角色"
+L.label_confirm_detective_only = "限制尸体的确认只限于警察角色"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1769]:
```suggestion
L.label_confirm_detective_only = "只允许警察角色确认死者"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 13]
```diff
@@ -1743,45 +1741,45 @@ L.label_prop_carrying_force = "Prop推进力"
...
+L.label_nade_throw_during_prep = "在准备时间内允许投掷手榴弹"
 L.label_postround_dm = "回合结束后启用死亡竞赛"
 L.label_spectator_chat = "启用观察者与大家聊天的功能"
 L.label_lastwords_chatprint = "如果在打字时被杀，则发出最后一句话至聊天室"
 L.label_identify_body_woconfirm = "不按'确认'按钮识别尸体"
 L.label_announce_body_found = "宣布发现了一具尸体"
 L.label_confirm_killlist = "宣布确认尸体时，该尸体的击杀名单"
---L.label_inspect_detective_only = "Limit corpse search to policing roles only"
---L.label_confirm_detective_only = "Limit corpse confirmation to policing roles only"
+L.label_inspect_detective_only = "将搜尸的范围限定为只限于警察角色"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1768]:
```suggestion
L.label_inspect_detective_only = "只允许警察角色搜查尸体"
```

File: lua/terrortown/lang/zh_hans.lua (hunk 11 above)
Comment (sbzlzh) [reply lua/terrortown/lang/zh_hans.lua:1065]: All right

File: lua/terrortown/lang/zh_hans.lua [hunk 14]
```diff
@@ -1743,45 +1741,45 @@ L.label_prop_carrying_force = "Prop推进力"
...
-L.label_firstpreptime_seconds = "首局准备时间(秒)"
-L.label_roundtime_minutes = "固定回合时间(分钟)"
+L.label_preptime_seconds = "准备时间（秒）"
+L.label_firstpreptime_seconds = "首局准备时间（秒）"
+L.label_roundtime_minutes = "固定回合时间（分钟)"
 L.label_haste = "启用急速模式"
-L.label_haste_starting_minutes = "急速模式开始时间(分钟)"
---L.label_haste_minutes_per_death = "Additional time in minutes per death"
+L.label_haste_starting_minutes = "急速模式开始时间（分钟）"
+L.label_haste_minutes_per_death = "每位玩家死亡的时间奖励（分钟）"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1757]:
```suggestion
L.label_haste_minutes_per_death = "玩家死亡给予的额外时间（分钟）"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 15]
```diff
@@ -1701,31 +1699,31 @@ L.help_item_armor_dynamic = [[
...
-观察者可以使用道具附身来附身于躺在世界中的道具,并使用缓慢充能的'飞击量表'来移动上述道具.
+观察者可以使用道具附身来附身于躺在世界中的道具,并使用缓慢充能的'飞击量表'来移动上述道具。
 
-'飞击量表'的最大值由一个基础值和其他判断值组成,其中击杀数/死亡数将影响该值.随着时间的推移,能量条会慢慢充电.设定的充电时间是为'飞击量表'中的一个点进行充电所需的时间.]]
---L.help_karma = "Players start with a certain amount of Karma, and lose it when they damage/kill teammates. The amount they lose is dependent on the Karma of the person they hurt or killed. Lower Karma reduces damage given."
---L.help_karma_strict = "If strict Karma is enabled, the damage penalty increases more quickly as Karma goes down. When it is off, the damage penalty is very low when people stay above 800. Enabling strict mode makes Karma play a larger role in discouraging any unnecessary kills, while disabling it results in a more “loose” game where Karma only hurts players who constantly kill teammates."
---L.help_karma_max = "Setting the value of the max Karma above 1000 doesn't give a damage bonus to players with more than 1000 Karma. It can be used as a Karma buffer."
+'飞击量表'的最大值由一个基础值和其他判断值组成,其中击杀数/死亡数将影响该值。随着时间的推移,能量条会慢慢充电。设定的充电时间是为'飞击量表'中的一个点进行充电所需的时间。]]
+L.help_karma = "人品值是用来减少无差别击杀的。玩家开始时有一定量的人品值，当他们伤害/杀死队友时就会失去人品值。他们失去的数值取决于他们伤害或杀死的人的人品值。较低的人品值会减少给予的伤害。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1705]:
```suggestion
L.help_karma = "玩家有一定量的初始人品值，伤害/杀死队友时会失去人品。失去的数值取决于他们伤害或杀死的人的人品值，对方人品越低，损失的人品越低。"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 16]
```diff
@@ -1701,31 +1699,31 @@ L.help_item_armor_dynamic = [[
...
+L.help_karma_strict = "如果更严格人品值被启用，伤害惩罚会随着人品值的减少而更快增加。当它关闭时，让人品值保持在800以上时的伤害惩罚是非常低的。启用严格模式使人品值在阻止任何不必要的击杀方面发挥更大的作用。而禁用它则导致一个更'宽松'的游戏范围，人品值只影响那些不断击杀队友的玩家。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1706]:
```suggestion
L.help_karma_strict = "如果严格人品模式被启用，伤害惩罚会随着人品值的减少快速增加。非严格模式下，人品值保持在 800 以上时的伤害惩罚非常低。启用严格模式使人品值在阻止任何不必要的击杀方面发挥更大的作用，而禁用它则导致一个更宽松的游戏范围，人品系统只会影响不断击杀队友的玩家。"
```

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by TheOnly8Z** [comment]
Changed some wording. Should be good to merge after these are fixed.

**Comment by sbzlzh** [comment]
Wait a minute, got it

**Review by TheOnly8Z — APPROVED** [review]
LGTM for zh_hans.

**Review by saibotk — APPROVED** [review]

**Review by TimGoll — APPROVED** [review]
Thanks a lot! Interesting to see that chinese doesn't seem to have whitespace chracters, does it?

**Pull request merged**